templates/terraform/environments/prod/main.tf

```diff
@@ -24,13 +24,13 @@ module "prod" {
   ecr_repositories = [] # Should be created by the staging environment
 
   # EKS configuration
-  eks_cluster_version      = "1.16"
+  eks_cluster_version      = "1.17"
```

Comment:
https://github.com/commitdev/zero-aws-eks-stack/blob/main/templates/terraform/modules/environment/main.tf#L30

can you help bump this to 0.0.2 as well

Reply:
Yes! Thanks! I knew there was something else.